mep/accounts/models.py

```diff
@@ -598,6 +598,15 @@ def save(self, *args, **kwargs):
             self.item_status = self.ITEM_RETURNED
         super(Borrow, self).save(*args, **kwargs)
 
+    def status(self):
```

Comment:
If this is just to get the display label for item status, there's a django method that gives us that for free: https://docs.djangoproject.com/en/1.11/ref/models/instances/#django.db.models.Model.get_FOO_display

Reply:
thank you for this!! I kept thinking "there must be a better way...."